lib_xua/lib_build_info.cmake

```diff
@@ -27,7 +27,7 @@ set(LIB_DEPENDENT_MODULES "lib_adat(1.2.0)"
                           "lib_spdif(6.1.0)"
                           "lib_sw_pll(2.2.0)"
                           "lib_xassert(4.2.0)"
-                          "lib_xud(2.3.1)")
+                          "lib_xud(develop)")
```

Comment:
Do we want to do this? or leave for now, release lib_xud then bump?

Reply:
The latest lib_xud has the RX rise delay change https://github.com/xmos/lib_xud/commit/039cd2b8d20f912cd17aa5ccb3e77c7b90227388 which is required for MIDI tests to pass on sw_usb_audio so I think we take the head of develop of lib_xud

Reply:
I've just made 2.3.2 release of lib_xud :)